misc/netgen/meta.yaml

```diff
@@ -31,6 +31,10 @@ requirements:
   run:
     - python
 
+test:
+  commands:
+    - netgen Q
```

Comment:
Looks like the test assume a `DISPLAY`, can we use `netgen -batch quit` instead?

Reply:
It works fine on my system. I think it only consults `DISPLAY` if compiled with X11.

Reply:
Yep, but the Linux build doesn't currently disable X11 it seems. So this makes the CI fail.

Reply:
Alright, let's see if that fixes it